### PR TITLE
Some general cleanup.

### PR DESCRIPTION
**Pull request by sfshaza2**
You don't both need to review it, but I hope one of you can.

Staged:  https://sz-flutter.firebaseapp.com/

### CODE REVIEW

File: src/docs/get-started/flutter-for/ios-devs.md [hunk 1]
```diff
@@ -200,13 +206,13 @@ Widget build(BuildContext context) {
...
+```
 
-You can add padding to any widget, which mimics the functionality of
-constraints in iOS.
+You can add padding to any widget,
+which mimics the functionality of constraints in iOS.
 
-You can view the layouts that Flutter has to offer in the [widget
-catalog](/docs/development/ui/widgets/layout).
+You can view the layouts that Flutter has to offer in the
```
Comment (legalcodes) [inline src/docs/get-started/flutter-for/ios-devs.md]: This wording left me with the impression that Flutter only has a limited number of layouts to offer.

**Comment by legalcodes** [comment]
IMHO this PR can be broken up into smaller PRs -- makes review easier, and also helps with tracking issues and fixes over time.

File: src/docs/get-started/editor.md [hunk 2]
```diff
@@ -37,12 +37,12 @@ VS Code. If you want to use a different editor, that's OK, skip ahead to the
 
 Android Studio offers a complete, integrated IDE experience for Flutter.
 
-* [Android Studio]({{site.android-dev}}/studio), version 3.0 or later
+* [Android Studio][], version 3.0 or later
```
[the remark this answers is not in the document]
Comment (sfshaza2) [reply src/docs/get-started/editor.md]: Not generally, but this is really a list with 3 items.

[the remark this answers is not in the document]
Comment (legalcodes) [reply src/docs/get-started/editor.md]: We're talking about the same part of the doc, right? FWIW this really looks like two separate lists to me:

Screen Shot 2020-02-21 at 11 26 49 AM

**Comment by sfshaza2** [comment]
Yes, we are. I still don't mind the lists, as they are. I wouldn't normally
do a one-item list, but this is parallel.

**Comment by sfshaza2** [comment]
@legalcodes, I'll tackle the diff2html in another PR.

**Comment by sfshaza2** [comment]
Hi, @legalcodes, aside from the diff2html, I saw only one remaining issue: WidgetApp => WidgetsApp. I searched the whole repo and found that error in 3 files. It's now fixed. Can we merge this?

**Comment by sfshaza2** [comment]
Grumble. I will fix this in a few minutes. I have another PR. Forcing a merge.

**Pull request merged**